src/components/Button/index.tsx

```diff
@@ -10,27 +11,31 @@ const Button = (props: TButtonPropTypes): JSX.Element => {
     size = 'large',
     className = '',
     iconProps,
-    buttonActionType,
+    buttonActionType = 'submit',
```

Comment:
do we need to have `static` as default value?

Reply:
done, thanks for good suggestion.